tpot/base.py

```diff
@@ -541,7 +533,15 @@ def predict(self, features):
         """
         if not self._fitted_pipeline:
             raise RuntimeError('A pipeline has not yet been optimized. Please call fit() first.')
-        return self._fitted_pipeline.predict(features.astype(np.float64))
+
+        features = features.astype(np.float64)
+
+        if self._contains_nan(features):
+            if self.verbosity > 1:
+                print('Imputing missing values in feature set')
+            features = Imputer(strategy='most_frequent').fit_transform(features)
```

Comment:
Maybe these 4 lines should be a function into themselves, since they're repeated >1 times?